lib/samson/hooks.rb

```diff
@@ -80,6 +144,16 @@ def hooks(name)
 
 Dir["plugins/*/lib"].each { |f| $LOAD_PATH << f } # treat included plugins like gems
 
-Samson::Hooks.plugins.
-  each(&:require).
-  each(&:add_migrations)
+module Samson::LoadDecorators
+  def inherited(subclass)
+    Samson::Hooks.fire(:model_defined, subclass)
```

Comment:
it's simpler if the interface is symmetric, setup takes a name, so fire should too